### DISCUSSION TITLE
Navigation notifications to smartwatch, espec Garmin, via Android notifcations channel.

### DISCUSSION THREAD

**Discussion by Runner11** [osmandapp/OsmAnd#12346] [Ideas]
Hi, I want my Garmin Forerunner 245 watch receive navigation hints sent by Osmand (espec while cycling). For that purpose I have activated in Garmin's smartphone-app the transmission of Osmand's notifications to the watch. Which actually works.
![IMG_20210802_120611_1](https://user-images.githubusercontent.com/88316094/127845760-9c20bace-6207-4cc0-b52f-249ebff1a83f.jpg)
However, way too many are sent, eg. counting down every 10 meters before a turn. Which obviously might make sense in the notification bar area of a a phone, but lesser as continously single alarm messages sent to a watch every other second.
Lotus offers a special app on the watch to which their smartphone-app connects to as a full blown solution (with map).
But I wonder if it would be possible to implement a much simpler solution: E.g. an option available in Osmand which says "Generate Android notifications for pre-turn and turn only. Avoid distance countdown".

As I'm still on Android 8, maybe sth has already been implemented, e.g. by message priorities or sth like that? Any hints welcome.

**Comment by djv** [comment]
I have the same problem on Android 16 with Forerunner 965.
